packages/compiler/core/checker.ts

```diff
@@ -1708,9 +1728,11 @@ export function createChecker(program: Program): Checker {
         if (base.flags & SymbolFlags.Alias) {
           base = getAliasedSymbol(base, undefined);
         }
-        const type = base.type ?? getTypeForNode(base.declarations[0], undefined);
-        lateBindMemberContainer(type);
-        lateBindMembers(type, base);
+        if (isTemplatedNode(base.declarations[0])) {
```

Comment:
This check just "happens" to align right? What matters is that its a late bind container? Or am I missing something?

Reply:
I think what I meant to check here is that I should be late binding this container and the only time we want to late bind something is if it is a template instance. Which this check isn't the most accurate for then. Let me double check if this shouldn't be tweaked and add more test cases(Like autocompleting a templated type declaration member)